src/main/java/at/hannibal2/skyhanni/utils/LorenzVec.kt

```diff
@@ -56,6 +57,13 @@ data class LorenzVec(
         return (dx * dx + dz * dz)
     }
 
+    operator fun plus(other: LorenzVec) = LorenzVec(x + other.x, y + other.y, z + other.z)
+    operator fun minus(other: LorenzVec) = LorenzVec(x - other.x, y - other.y, z - other.z)
+    operator fun times(other: LorenzVec) = LorenzVec(x * other.x, y * other.y, z * other.z)
+    operator fun times(other: Double) = LorenzVec(x * other, y * other, z * other)
+    operator fun div(other: LorenzVec) = LorenzVec(x / other.x, y / other.y, z / other.z)
+    operator fun div(other: Double) = LorenzVec(x / other, y / other, z / other)
```

Comment:
times(LorenzVec) & times(Double) and div(double) we already have. Rename them instead of doubling the functions

Reply:
These are different functions from the already existing `multiply`, `add`, `divide`, and `subtract` these are operator overloading so instead of `vec1.add(vec2)` it can now be `vec1 - vec2` these ones are far easier to use than functions.

Reply:
aware, but the old ones shouldn't exist

Reply:
The old functions are used by 100s files removing the old ones would cause many merge conflicts with already opened PRs, the functions should just be deprecated instead, and new features and changes should edit them overtime as to not cause major conflicts between PRs.

Reply:
yeah, exactly what I meant (implicitly). Cal knows how to handle this (I assume) therefore I wasn't where specific, only point out the fact that we have duplicates. Maybe I should have written the original message better.